client/views/edit/edit.js

```diff
@@ -0,0 +1,168 @@
+import { Instances } from '/lib/common.js';
+
+function showEditAdvError(reason) {
+  if (typeof currentError !== 'undefined') {
+    Blaze.remove(currentError);
+  }
+  const parentNode = document.getElementsByClassName('formcontainer')[0];
+  const nextNode = document.getElementsByClassName('inputcontainer')[0];
+  currentError = Blaze.renderWithData(Template.form_error, reason, parentNode, nextNode);
+}
+
+Template.edit.onRendered(() => {
+  // When the template is rendered, set the document title
+  $('.formcontainer').hide().fadeIn(400);
+  $('#darker').hide().fadeIn(400);
+});
+
+Template.edit.helpers({
+  // threshold
+  thresholdFirst() {
```

Comment:
These can all be collapsed into a single `isSelected` function that takes two arguments, like we talked about.